spec/system/user_spec.rb

```diff
@@ -1,18 +1,19 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
+require 'spec_helper'
 
-describe User do
```

Comment:
You may want to rename this to match a feature instead of an object like we do with our feature tests. I know I'm the one who named it but I wasn't thinking about what these tests were actually for.

Reply:
You're commenting on code I deleted. I totally did what you suggested.